src/proto/grpc/testing/messages.proto

```diff
@@ -202,8 +202,13 @@ message LoadBalancerStatsRequest {
 }
 
 message LoadBalancerStatsResponse {
+  message RPCsByPeer {
+    // The number of completed RPCs for each peer.
+    map<string, int32> rpcs_by_peer = 1;
+  }
   // The number of completed RPCs for each peer.
   map<string, int32> rpcs_by_peer = 1;
   // The number of RPCs that failed to record a remote peer.
   int32 num_failures = 2;
+  map<string, RPCsByPeer> rpcs_by_type = 3;
```

Comment:
This should probably be called `rpcs_by_method`.

I assume this field name is already being used in other languages?  If so, it's probably not worth changing.

Reply:
It is used, but nothing has been submitted.
I can change them if you prefer that.

Reply:
Updated to `rpcs_by_method` in the other PRs.